src/test/java/com/google/api/codegen/transformer/RetryDefinitionsTransformerTest.java

```diff
@@ -180,18 +160,16 @@ public void testWithInterfaceOnly() {
     Map<String, ImmutableList<String>> retryCodesDef = retryCodesConfig.getRetryCodesDefinition();
     Map<String, String> retryCodesMap = retryCodesConfig.getMethodRetryNames();
 
-    Truth.assertThat(retryCodesMap.size()).isEqualTo(4);
+    Truth.assertThat(retryCodesMap.size()).isEqualTo(3);
```

Comment:
`assertThat(retryCodesMap).hasSize(3)` it generally gives better error message like `map [a->b x->y] has size 2 but want 3` instead of `not true that 2 equals 3`

Reply:
wow google's Truth library has everything you could ever want to assert...
done!